inspirehep/modules/records/serializers/schemas/base.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Schema for parsing literature records."""
+
+from __future__ import absolute_import, division, print_function
+
+from pybtex.database import Entry, Person
+from ..fields_export import get_authors_with_role, extractor, bibtex_type_and_fields
+from inspire_utils.record import get_value
+from inspirehep.modules.records.json_ref_loader import replace_refs
+
+
+class PybtexSchema(object):
+    def load(self, json):
+        """
+        Args:
+            json: literature record from API
+
+        Returns:
+            Pybtex Entity representing the record
+        """
+        json = replace_refs(json, 'es')
+        doc_type, fields = bibtex_type_and_fields(json)
+        texkey = unicode(get_value(json, 'texkeys[0]'))
```

Comment:
you should use `six.string_type` instead of `unicode` for Python 3 compatibility if this is really needed, but usually `str` -> `unicode` is transparent (if only ASCII is used like here).